## Limitations {#sec:limitations}

Although the strategies for elucidating the emergence of complex traits discussed in this thesis are promising, there are practical limitations that cannot be circumvented; at least not easily. In cases where natural populations are studied, you are dependent on capturing the right moment when the biochemical process you are interested in is active. If you don't sample at the critical time point, you risk to miss the processes that actually defines the trait of interest. In the context of gene expression, the half-life of mRNA is relatively short, and

Sample sizes. In order to have enough power in association studies, the larger the sample size the more power you have. The problem when using forest trees is that it is very expensive and time consuming to maintain a large population of trees. Ideally, the trees would be grown in a common garden setting in order to control for environmental factors, but it is not feasible to do this in the same way that is done for human studies, such as the example of human height were more than 250,000 samples were used [@Wood2014]. A common garden experiment comprising 250,000 trees will simply not happen. Of course, these could be sampled directly in the field, but in order to sample this, researchers would have to physically get out and sample this number of samples across the environmental range for the study. Environmental factors would then have to be controlled for in some other way, and given that environmental information like this is not widely available, this is not an easy task. Furthermore, you cannot ask a tree how old it is.

When it comes to geographical range, you are limited to the range that you have sampled, and you cannot do any kind of inferences beyond that. For example, in papers III and IV, we were limited to the range of Sweden, while aspens are spread out across more or less the whole northern hemisphere. In that context, Sweden is a very small part of that, and it might not be very surprising that we did not see any pronounced population structure in the data.

The importance of good annotations and previous efforts to elucidate regulatory mechanisms also play a big role. This is something that is touched upon in @Mathelier2015. As of yet, most of these efforts have been directed towards human studies, quite understandably. While the results from these studies are not directly transferable to other species, it can still be used as a rough guide to how things look in humans when it comes to a particular phenomenon. Then it will also be even more interesting if the results do not match what has been found previously. This could then either be due to a mistake, but it could also be biological mechanisms that we simply do not observe in humans [REFS EXAMPLES].

- Gene expression and phenotypic sampling is limited to the moment that you sample. The half-life of mRNA is relatively short, and it varies throughout the year, and even throughout the day. If you sample at the wrong time, you might miss the signals that are of significant for the particular trait you are interested in.
- Rare variants and information content
- Low effect variants: how many samples do you need? Human height study with 200k samples.
- Even though I've been saying that the regulatory genome is a big thing, a lot of annotations are still focused on coding genes. In order to fully understand gene expression regulation, non-coding genes must get better annotations, and they must also be included in eQTL mapping efforts.
- Missing heritability: in most cases only a part of the phenotypic variation can be explained by genetic variants and environment. The rest is referred to as missing heritability. In reality this heritability is of course not missing; it is just due to factors that we cannot observe for some reason or another. For example, there might be rare genetic variants that affects the phenotypic outcome that we are not able to detect without a very large sample size. In some cases, it might even be the case that the whole global population would not be enough to find a statistically significant relationship for some variants. Also, epistatis plays a big role in this as well, and is also very hard to outline because of the huge number of possible combinations of genetic variants.
- Some limitations are brought up in [@Mathelier2015]: the importance of good annotations and previous rigorous studies. Model organisms that have been studied for a long time have a huge advantage here. This is not saying that the information from these studies cannot be transferred to other species, but they will need a proper breakdown in other species as well. Hopefully the studies in e.g.\ humans can help other areas as well.
- As discussed in several sections of this thesis, most complex traits are the result of many interacting factors. Trying to exhaustively search through combinations of e.g.\ SNPs quickly leads to computations that will not finish before the sun goes dark (given the computational power of today). For example, in paper III, 3.2 million SNPs and about 20,000 genes were considered for eQTL mapping. If we wanted to investigate the combinatorial effects of SNPs on gene expression by considering all pairs of genes acting upon each gene, the number of associations would go from 64 billion (which already is a huge number) to about $1.02 \times 10^{17}$ tests. Extending this to combinations of three SNPs results in $1.09 \times 10^23$ tests. Assuming that we could calculate 1 million models per second, all combinations of two SNPs would take xxx hours, or xxx years. All combinations of three SNPs would take xxx hours or xxx years.
