## Association studies

Association studies, in this context, refers to the association of genetic variants with a phenotype of interest. This phenotype can range from very clear ones, such as human height, to more abstract phenotypes, such as gene expression. In the following sub-sections I will present the concept of association studies and what we have learned from those so far.

### Genome-wide association

I dare to bet that most people have come into contact with genome wide association studies (GWAS) at one time or another. Whenever you see headlines in the news such as "the obesity gene has been found", it is likely that the underlying study is a GWA study. It is also likely that the sensational headline is not quite true. What researchers have done in cases like this is to collect populations of individuals; those that have the phenotype of interest, such as a disease, and another population of healthy people. The genomes of these individuals are then sequenced or otherwise assayed for genetic variants and the researchers then ask themselves: can we identify variants that can be used to tell the difference between healthy and diseased individuals?

This is usually performed using relatively simple regression models where the disease status or the quantitative phenotype acts as the response variable and the genetic variants as the explanatory variable ([@fig:genetic-association]). A straight line is then fitted to this data which then will represent the predicted trait value for a given genotype ([@fig:genetic-association]). This assumes that there is an additive effect, that is, theIt is not hard to imagine that this can potentially results in a huge number of tests. In humans, for example, we expect to find one SNP every 1.9 kilobases in the genome [@Sachidanandam2001], and this would result in more than 1.5 million tests if one were to test the association to the phenotype of interest for each and every genetic variant. This has a few consequences, but mainly it requires computational power that has only gotten widely available quite recently and the multiple testing burden becomes quite heavy.

Multiple hypothesis testing is a statistical problem where we expect to see random associations when performing a large number of tests. In the case of GWA in humans, let's say that we are associating 1.5 million variants with a particular phenotype. Applying the commonly used p-value threshold of 0.05 means that we would expect 75,000 false positive associations, i.e.\ associations that are purely random simply due to the number of tests performed. To control for this the p-values can be adjusted according to the number of tests performed with something like the Bonferroni correction where the obtained p-values are multiplied by the number of tests performed. Depending on the context, other less stringent methods are often preferred, like the false discovery rate (FDR) [@Benjamini1995]. In studies involving genomic and gene expression data, there is extensive correlation structures in the data. Using relatively simple approaches such as Bonferroni or FDR correction will not take the correlation structure into account, and this can be overcome by using permutation tests.

In order to detect associations with a very low effect, i.e.\ a slope close to zero to connect to @fig:genetic-association, a large number of samples is needed. One problem with using a large number of samples is that a lot of spurious associations will be detected as well. With fewer samples, only the most obvious associations will be detected, i.e.\ those with a high effect. For example, in the study of human height mentioned previously, more than 250,000 individuals were included in the study [@Wood2014], and a meta-analysis of almost the same magnitude was performed to find a genetic explanation to body mass index [@Speliotes2010], and in both cases the phenotypic variance explained by individual variants was very low (below 1%).

![A schematic example of an association study where three different individuals with three different genotypes at a particular locus (left): A/A, A/G, and G/G. These genotypes explain the height of these individuals where the G allele is associated with higher individuals. Associating this locus with the height of individuals in a population might yield the plot to the right. The dashed line is then fitted to the data to minimise the distances between all data points and this line. If the slope of this line is significantly different from zero we say that the association is significant.](figures/associations.pdf){#fig:genetic-association width=100%}

### eQTL mapping {#sec:eqtl-mapping}

Expression quantitative trait locus (eQTL) mapping is related to the traditional GWAS in the previous section, but the phenotypes here are of the more abstract kind; something you cannot observe directly, namely gene expression. Also, the problems of GWAS get even bigger for this type of association study since not only do we have a large number of genetic variants, we also have a large number of phenotypes. The phenotypes in this case is gene expression. If we are to consider the expression of every gene in the genome together with all the genetic variants in the genome we get, for humans, approximately 30 billion tests have to be performed. Not only does this results in a multiple testing problem, but it also causes purely computational problems. Not too long ago, this would have been practically impossible to perform due to the computational resources needed, but with the increase in computational power coupled with clever methods [@Shabalin2012] this is now relatively easy to do.

Traditionally, QTL mapping can divided into two categories: linkage mapping and association mapping. Linkage mapping is usually used when family information is available, such as in a controlled cross, and relies on known markers. By performing a cross and observing how genetic markers associate with changes in the trait of interest. This is not something that has been done during the projects associated with this thesis, due to a couple of reasons. Firstly, dealing with natural populations of plants typically means that you do not have family information, and secondly, known genetic markers are few and far between in species such as aspen. This is where association mapping, or linkage disequilibrium mapping, comes into the picture. This method is related to GWA in that a large number of genetic markers (typically SNPs) are associated to a phenotype; the phenotype in this case being gene expression levels.

eQTLs can be divided into being either local or distant. A local eQTL is close to the gene that it is associated with while a distant is far away on the same chromosome or on a different chromosome than the associated gene. The distance where local becomes distant is however somewhat arbitrar. In our eQTL analysis in paper III, we classify anything within 100 kilobases from the transcription start site to be local based on the distance distribution of eQTL on the same chromosome as the associated gene. The division into local and distant is a purely structural as opposed to functional definition. A more functional definition exists as well where eQTLs are divided into two groups depending on *how* they act on the associated gene. eQTLs are said to act either in *cis* or in *trans*, with *cis*-eQTLs acting directly on gene expression while *trans*-eQTL act indirectly on the associated gene. An example of a *cis* mechanism could be a variant that modifies a transcription factor binding site in the promoter of a gene, while a *trans* effect could be something so subtle as affecting the abundance of a certain co-factor that is needed to express the associated gene. Based on these examples, we can see that a *cis*-eQTL should act in an allele specific manner. If a transcription factor binding site gets disrupted in only one allele, only the transcription of that allele will be affected. Conversely, *trans*-eQTLs will have the same effect on both alleles. Due to the indirect mechanism of *trans*-eQTL, these are generally of lower effect (remember the slope from [@fig:genetic-association]), and this is something that has been reported by numerous studies (references in [@Breitling2008; @Kliebenstein2009]), but there are exceptions [@Clement-Ziza2014]. The case is often that *cis* acting variants are local to the associated gene while *trans* effects are more distant. Many studies have opted to only consider *cis*-eQTL, and this is to some extent a tactical decision in that it makes the computational problem a bit easier since fewer tests have to be performed, and consequently, the multiple testing problem becomes slightly less of a problem since the number of markers considered for each gene is much smaller than the total number of markers.

The first study of the genetics underlying gene expression variation was performed in yeast in 2002 [@Brem2002] and included 3,312 genetic markers and 6,215 genes. At the time this was a big feat, but today we are able to run association tests for all genes in the genome and all markers as demonstrated by the Genotype Tissue Expression project (GTEx; [@Ardlie2015]) with a total of about 6.8 million SNPs and using both coding and non-coding genes (53,934 genes in total).

#### Biology gets complicated quickly {#sec:complexity}

Complex traits are the result of the interactions between many different factors. Often this results in redundancy in the regulatory network, redundancy that can act as a buffer for random mutations [@Rutherford2000]. Here gene duplications play a role as well since with two copies of the same gene, any detrimental mutations to one of them will most likely not affect the organism in a drastic way. Not only does this protect the organism, but it can also hide the regulatory mechanism from traditional analysis methods. For example, when it comes to eQTLs, the traditional way of doing it is to consider pairs of genes and genetic variants one by one. A better approach would be to analyse combinations of genetic variants and how they affect gene expression in concert. However, it is not possible to do this in an exhaustive manner due do computational complexity. In [@fig:complexity] a simple example of how the regulation could be hidden from traditional analysis methods. In this case, the regulation of G3 could only be dissected if three SNPs were taken into account in the model that explains the expression of G3. Any single one of these SNPs will not be able to explain the expression by themselves, but together they explain it perfectly. This is a very simplified example, but it highlights the inherent difficulties of systems genetics. In order to dissect the regulation of R3 without any prior information, the model would go from one SNP per gene to two SNPs per gene, i.e. looking at how all pairs of SNPs explain the gene expression. In paper III we work with about 3.2 million SNPs and about 20,000 genes resulting in about 64 billion models. To capture the expression of R3 we would have to create models using all pairs of SNPs against all genes and this would result in $1.02 \times 10^{17}$ models. The expression of G3 in [@fig:complexity] is explained by three eQTLs, and in order to test all SNP triplets, resulting in $1.09 \times 10^{23}$ models. Assuming that we are able to calculate 10 million models per second---which is about the same speed as we achieved in paper III---computing all models for pairs of SNPs would take more than 300 *years*, and all models of SNP triplets would take more than 34 *million* years.

One way to think of this is that simplicity would be bad for biology in general. If something is easy to disentangle, then a very small perturbation, like a mutation, could possibly disrupt a whole system. This is part of why we, in paper III, hypothesise that genes that are central in the co-expression network have more redundancy in their regulation. By having more redundancy, these genes will not be affected as easily by random mutations, and this is the same idea underlying the hypothesis of scale-free biological networks ([@sec:coexpression]).

![Simplified example of when eQTL effects and gene regulation is masked. In the regulatory network, the regulators R1 and R2 are always on, and regulator R3 is on as long as not both of the eQTLs Q2 and Q3 block the signal. The expression of G1 only depends on Q1, and this eQTL is thus detected since there is a perfect relationship between the genotype and the expression. Due to the dual regulators and eQTLs for R3, there is no perfect relationship between the eQTLs Q2 and Q3 with either R3 or G2. The regulation of G3 is even more complicated where R3 needs to be expressed, and at the same time Q4 must not block the signal. No perfect relationship between G3 and any of the eQTLs exist even though Q4 is *cis*-acting and Q2 and Q3 are both *trans*-acting.](figures/complexity.pdf){#fig:complexity width=100%}
