# Discussion

## Future perspectives

- Cheaper sequencing costs, new sequencing technologies such as Oxford Nanopore that generates much longer reads compared to the current generation of high throughput sequencing machines, such as Illumina HiSeq. Longer reads mean more accurate expression estimates, and resolution at the transcript scale, meaning we can separate between splice variants of the same gene that potentially can have different function.
- From the above: Splice QTLs (http://melissagymrek.com/science/2014/11/20/splice-qtls.html)
- CRISPR/Cas9: The possibility to edit the genome in order to track down the causal variants, and also the possibility to verify potential regulatory variants.
	- GWASs are limited. They are simply statistical associations.
	- Gene by gene approaches will not be feasible/possible when a trait is epistatically controlled. This will be possible using CRISPR/Cas9.
	- Another possibility with this is to perfom allele specific changes.
	- There is already software available for designing guide RNAs that are used for the actual genome editing process (?) [@Heigwer2016; @Chari2015; @Heigwer2014; @MacPherson2015b; @Montague2014; @Wong2015].
	- Great resource on the history and the possibilities with CRISPR/Cas9 on https://www.addgene.org/crispr/guide/.
- The ultimate goal would be to be able to say that if we change this base from an A to a G, we will modify the regulation of thisnthat and get the exact phenotype that we want.
- With this comes an avalanche of ethical dilemmas.

- Populations and a single reference genome. There is no ground truth. Reference genomes must become more flexible and allow for genetic variants. Today, we see bias in gene expression quantification where reads from RNA-Seq reads are aligned to a reference genome [REF]. If there is a polymorphism in the studied sample that is not present in the reference genome, the expression of that gene will be underestimated [REFS]. Reference allele bias.

- Catch 22 when it comes to small effects. We need large sample sizes in order to detect them, and when we have detected them, we instead have a huge combinatorics problem that we simply cannot solve without any prior information.

- How can these types of studies help understand the emergence of natural variation?
- Talk about the potential for applications for this technology, i.e.\ new tree/crop variants that can handle climate change; drought, salinity, improved traits
	- With this the mandatory discussion about GMO resistance comes as well. I should say a few words about it, but don't linger on it.

- These are exciting times that we are living in. On the other hand, looking back at the historical examples in this thesis, I am sure that this sentence has been uttered in each and every one of these. In short, biology is exciting!
