## Limitations {#sec:limitations}

Although the strategies for elucidating the emergence of complex traits discussed in this thesis are promising, there are practical limitations that cannot be circumvented, at least not easily.
In cases where natural populations are studied, we are dependent on capturing the right moment when the biochemical process of interest is active.
If we don't sample at the critical time point, we risk missing the processes that actually define the trait of interest.
Not only is the sampling time point critical for capturing e.g.\ a developmental stage, it is also important to consider daily fluctuations that can have an effect on gene expression.
This is clearly demonstrated in a recent study in rice where gene expression was quantified in the field, and it was shown that even short term variations in temperature and sunlight levels modified gene expression in a reproducible manner [@Plessis2015].

When it comes to geographical range, we are limited to the range of sampling, and cannot do inferences beyond that.
For example, in papers III and IV, we were limited to the range of Sweden, while aspens are spread out across more or less the whole northern hemisphere.
In that context, Sweden is a very small part of the total distribution range, and it might not be very surprising that we did not see any pronounced population structure in the data.

In order to have enough statistical power in association studies large sample sizes are needed.
A problem with using forest trees is that it is very expensive and time consuming to maintain a large population of trees.
Ideally, we want to grow them in a controlled environment in order to minimise environmental effects, but this is clearly not a practical approach.
In the study of human height that has been mentioned several times in this thesis, they used more than 250,000 individuals, but this was a meta-study [@Wood2014].
Even so, a meta-study of the same magnitude in *P.\ tremula* is not possible today, since the amount of data generated is not even close to that of human studies.
The importance of good annotations such as previous efforts to elucidate regulatory mechanisms also play a big role [@Mathelier2015].
As of yet, most of these efforts have been directed towards human studies (e.g.\ ENCODE [@Feingold2004]), quite understandably.
While the results from these annotation efforts are not directly transferable to other species, information regarding general characteristics of genome structure and function can most likely be transferred to other species.
This has to some extend been done already, but in the other direction.
For example, the fruit fly *Drosophila melanogaster* has been used as a model organism for human genetics and disease for over a century [@Stephenson2013].

The results of associations studies are just that---associations.
A genetic variant that is associated with a particular phenotype is not necessarily the causative variant.
It might be that it in turn is associated with the causal variant through linkage disequilibrium (LD), i.e.\ the non-random association of alleles at different loci.
In the case of plants a variant in LD with the causal variant might be good enough in many cases where marker assisted selection can be employed in breeding.
However, if more control of the phenotype is needed, a variant in LD is not of much help.
If the variant is no causal, it is likely that mutating this position will not result in any phenotypic change.
Strategies to filter out the causal variants from association studies include integrating different types of data in order to single out the most likely candidate genes or loci, but also transparency in when it comes to publication in order to minimise confirmation bias and consequently the number of false positives in circulation [@MacArthur2014].
