# Discussion

## Future perspectives

- Cheaper sequencing costs, new sequencing technologies such as Oxford Nanopore that generates much longer reads compared to the current generation of high throughput sequencing machines, such as Illumina HiSeq. Longer reads mean more accurate expression estimates, and resolution at the transcript scale, meaning we can separate between splice variants of the same gene that potentially can have different function.
- From the above: Splice QTLs (http://melissagymrek.com/science/2014/11/20/splice-qtls.html)
- CRISPR/Cas9: The possibility to edit the genome in order to track down the causal variants, and also the possibility to verify potential regulatory variants.
- The ultimate goal would be to be able to say that if we change this base from an A to a G, we will modify the regulation of thisnthat and get the exact phenotype that we want.
- With this comes an avalanche of ethical dilemmas.

- Populations and a single reference genome. There is no ground truth. Reference genomes must become more flexible and allow for genetic variants. Today, we see bias in gene expression quantification where reads from RNA-Seq reads are aligned to a reference genome [REF]. If there is a polymorphism in the studied sample that is not present in the reference genome, the expression of that gene will be underestimated [REFS]. Reference allele bias.

- Catch 22 when it comes to small effects. We need large sample sizes in order to detect them, and when we have detected them, we instead have a huge combinatorics problem that we simply cannot solve without any prior information.
