## Gene expression {#sec:gene-expression}

Genetic variation does not really have any significance if it does not manifest itself in some kind of way. One way that it can manifest itself is through gene expression. Genes are pieces of DNA that are transcribed into messenger ribonucleic acid (mRNA), and eventually translated into proteins. This is a very simplified view of what is going on. In every step of the process there are different forks in the road that can be taken, and each of these forks will change the fate of that gene. It can lead to it eventually being translated to protein, or it might result in splicing out part of the gene that gets translated to a different protein, or it could lead to the degradation of the mRNA, to mention a couple of examples. Also, this is a highly dynamic process that responds to different kinds of stimuli, such as environmental changes.

![Schematic overview of gene expression. The transcriptional machinery including the RNA polymerase (1) is recruited to the promoter of the gene to be expressed by transcription factors (2) that bind to the promoter and any enhancers (3) through specific motifs in the DNA (4). The DNA is then translated to RNA by the RNA polymerase (4) and introns are spliced out (5) before the mature mRNA is translated into protein. This is a simplified view of how coding regions are transcribed.](figures/transcription.pdf){#fig:gene-expression width=100%}

### Regulation of gene expression {#sec:regulation-of-gene-expression}

As stated in @sec:genetic-variation, the regulatory genome has gained more and more attention in the last 15 years. New studies keep showing examples where protein sequence is identical between vastly different phenotypes, but changes in gene regulation is instead what is responsible for the phenotypic variation. Examples of this are beak length [@Abzhanov2006] and beak shape [@Lamichhaney2015] in Darwin's finches, and the previously mentioned differences between humans and chimpanzees [@Pollard2006; @Polavarapu2011]. In the study of Darwin's finches by @Lamichhaney2015, one of the genes that was associated with the differences in beak shape was a transcription factor (TF). Transcription factors are proteins that bind to promoter regions upstream of genes and consequently recruits the transcriptional machinery involving the RNA polymerase ([@fig:gene-expression]). The regions that transcription factors recognise are commonly referred to as motifs. They are short DNA sequences with a specific composition that is meant to match the binding residues of a transcription factor. These motifs are often degenerate and can thus be difficult to detect [@Stormo2000], and it is very difficult to predict the effect of a single mutation in one of these binding sites. One recent study characterised the effect on gene expression by somatic mutations in cancer tumours and found that a lot of the genes that had their regulation modified by mutations in transcription factor binding sites [@Mathelier2015]. This study emphasises the important role that the regulatory genome plays in complex disease.

Transcription factors play one role in the regulation of gene expression. They can either activate or repress gene expression, and in many cases several transcription factors, both activators and repressors, are involved in the regulation. These types of combinatorial relationships makes it very difficult to try out all combinations of regulations in order to explain the expression patterns of a gene. One example of a study that tries to dissect the combinatorial nature of gene expression regulation is [@Street2011] where the authors are able to explain gene expression inside transcriptional modules based on the expression of the regulators. In this case, the dimensionality problems associated with these kinds of studies was reduced by firstly limiting the number of studied genes by only considering those that were expressed in a particular tissue. Furthermore, the regulators considered for each module was determined in an iterative fashion where a new regulator only was added if it increased the predictive power of the model.

Most of the genes get transcribed at some point in the lifetime of an organism, but this could possibly be at a single time point in a specific tissue. Based on this, it is easy to see that there must be a very complex regulatory system behind this. Transcription factors have to be expressed, and they in turn regulate the expression of some other gene that in turn might act in a feedback loop to regulate the expression of itself. This quickly turns into a complex network that is not easy to disentangle. To further complicate things, even if all factors needed to transcribe a particular gene are available, the gene might still not be expressed if the 3D structure of the DNA is not arranged properly. In order for e.g.\ enhancers to act properly, they need to be physically close to the gene it acts upon ([@fig:gene-expression]). Due to this complexity most studies of regulatory networks have so far been limited to a smaller sets of genes [@Chai2014].

### Quantifying gene expression

Similarly to genome sequencing, the estimation of gene expression has gotten a big boost by the development of high-throughput sequencing. In the case of gene expression, instead of extracting and sequencing the DNA, the mRNA is extracted and sequenced---RNA-Sequencing. This effectively creates a snapshot of the total mRNA abundance---the transcriptome---in the tissue of interest at the time of extraction. The last part of this sentence is something that is very important to consider, and I will come back to this in [@sec:limitations]. With the previously very popular microarray technology, this was also possible, but limited to the genes that were included on the array. With RNA-Sequencing, *all* mRNA in the cell can, theoretically, be sequenced, regardless of whether the gene expressing it is known or not.

The data from RNA-Sequencing is similar to that from DNA-Sequencing in that it consists of sequence reads based on a template sequence. In order to quantify gene expression these reads are aligned to either a reference genome, or a reference transcriptome. In addition to being able to measure the expression of all genes and not only known genes, the dynamic range is significantly wider in RNA-Sequencing compared to microarrays since the signal does not get saturated, and the noise levels are lower [@Wang2009]. The reads are aligned to a reference transcriptome (all known mRNAs in the organisms) or a reference genome (all DNA). Both approaches have their advantages and disadvantages, but perhaps the most obvious is that only known gene products will be detected if the reads are aligned to a reference transcriptome. Aligning to a reference genome means that no prior information about known genes is used; if reads map to an unannotated region of the genome, then there is something that is expressed in that region. One problem of aligning to the genome is that splice junctions have to be handled. A splice junction is the border between an exon and an intron, and these are not present in a mature mRNA ([@fig:gene-expression]).

A problem that exists for both of these methods is multi-mapping reads---reads that map to multiple locations in the genome/transcriptome. Duplicated genes for example might show this behaviour. Even if the genes have diverged when it comes to their regulation, their coding sequences can be more or less identical, and you cannot confidently say where this read originated from.

Another problem slightly related to multi-mapping reads is that of alternative splicing variants. These are mRNAs that come from the same gene, but they have different composition of exons. Some variants might be missing an exon that other transcripts have, for example.
One possible way around this problem is to look at the expression of exons, and not genes or transcripts as a whole [@Anders2012]. Another more recent approach to this is pseudo-alignment as done in software such as sailfish [@Patro2014] and kallisto [@Bray2016]. The principle of pseudo-alignment is to not care about the exact location of every read, but only conclude that the read belongs to a particular transcript. These types of methods are quite new and have not been tested enough yet, but they are still very interesting due to their speed; kallisto is 150--350 times faster than software that traditionally has been used for quantifying gene expression in RNA-Seq data [@Bray2016].

### Co-expression networks {#sec:coexpression}

When two genes have similar expression profiles they are said to be co-expressed. Co-expression can be seen as a manifestation of the underlying regulatory network---if two genes are regulated by the same factors, chances are that these genes also are co-expressed. In contrast to the regulatory network, the co-expression network is very simple to construct where the simplest approach would be to calculate the correlation between all pairs of genes. A co-expression network can be represented as a graph structure where the vertices are genes and the edges represent the degree of co-expression ([@fig:co-expression]). The consensus from a large number of studies is that co-expression networks (and biological networks in general) often are scale-free. What this means is that there are few genes with many connections to other genes (high degree centrality) in the network and more genes with few connections (low degree centrality) [@Barabasi1999].
If the network is disturbed by random perturbations, such as mutations, genes with a high degree centrality are less likely to be targeted, due to their low frequency in the network [@Whitacre2012]. Consequently, a scale-free network will be robust against random perturbations.
Another measure of centrality is betweenness centrality which is a measure of how often a node is part of the shortest paths between all pairs of nodes in the network ([@fig:co-expression]).
Co-expression networks have also been found to be modular, that is, there are sub-networks in the global network that are more tightly connected to the inside of the module than to the outside. More often than not it is the case that these modules are enriched in functional categories such as Gene Ontology [@Ashburner:2000ja] terms or Kyoto Encyclopedia of Genes and Genomes [KEGG; @Kanehisa2000] pathways [@Langfelder2008; @Langfelder2011].

![If the expression of two genes is correlated they are said to be co-expressed. This can be represented as a graph, or network, structure where each node represents a gene, and the edges between nodes represent significant co-expression. Modules in the network are defined as sub-networks that have a stronger connections to genes inside the module compared to genes outside the module. The modules are here represented by node colour.](figures/coexpression.pdf){#fig:co-expression width=100%}

Studies have shown that co-expression networks are able to capture biologically relevant gene expression signatures. One example is a study performed in lake whitefish (\textit{Coregonus clupeaformis}) where researchers found network modules that were correlated with dwarfism among these fish [@Filteau2013]. Another example is a cancer study where gene expression signatures common across tumour types were identified using a co-expression network approach [@Doig2013].

As previously stated, the co-expression network is a manifestation of the underlying regulatory network, but it is also important to remember that the co-expression network is a very simple representation of the correlation in gene expression levels. It only captures the state of mRNA in the cell at the time of sampling, and it is not necessarily a representation of the corresponding protein abundance. Two proteins that *can* interact *will* not necessarily interact just because their corresponding transcripts are expressed at the same time [@Vogel2012]. However, if two genes are expressed simultaneously, it might be the case that they share at least part of their regulation. One study exploring this in *Arabidopsis* used network cliques---sub-networks where all nodes are connected to all other nodes---to identify potential transcription factor binding sites [@Zheng2011Large].
